src/effects/effectparameter.h

```diff
@@ -39,6 +39,7 @@ class EffectParameter : public QObject {
     ///////////////////////////////////////////////////////////////////////////
 
     EffectManifestParameter::LinkType getDefaultLinkType() const;
+    EffectManifestParameter::LinkInversion getDefaultLinkInversion() const;
```

Comment:
I have no idea why all of the get/set functions in here are `virtual`... Why would someone want to override a get/set method? That's why I thought you were overriding it here, because otherwise it doesn't make sense to use the `virtual` keyword. I know you're just blending in with the rest of the code in the class, but just because the rest of the code is a bit sloppy, it doesn't mean that you should follow suit IMO.